DMX.Gatekeeper.Api.Tests.Acceptance/APIs/Labs/LabApiTests.cs

```diff
@@ -38,104 +39,7 @@ public LabApiTests(DmxGatekeeperApiBroker dmxGatekeeperApiBroker)
         private static int GetRandomPowerLevel() =>
             new IntRange(min: 0, max: 101).GetValue();
```

Comment:
If we aren't going to create the LabDevices then we don't need this anymore

Reply:
@terrypalo great catch. Removed along with the other methods.